variables.mk

```diff
@@ -167,6 +167,7 @@ PERMISSIVE_OFF=+permissive-off
 BINARY ?=
 override SIM_FLAGS += +dramsim +dramsim_ini_dir=$(TESTCHIP_DIR)/src/main/resources/dramsim2_ini +max-cycles=$(timeout_cycles)
 VERBOSE_FLAGS ?= +verbose
+RANDOM_SEED ?= 1
```

Comment:
I think we should have this actually be a random number most of the time.
And when the user suspects nondeterminism they would fix the seed to a particular value.

This is assuming the current behavior has a random seed.

Reply:
The current behavior in VCS uses 1 as the seed for every run. I think having default be deterministic makes more sense, otherwise it will always be annoying to reproduce some non-determinism-induced bug.

Users looking for randomness can just set `RANDOM_SEED=$RANDOM`